### PR TITLE
Fat 3417 nolana autotest's fix

### PR DESCRIPTION
**Pull request by penfriend**

### CODE REVIEW

File: cypress/support/fragments/circulation/newNoticePolicyTemplate.js [hunk 1]
```diff
@@ -40,7 +40,9 @@ export default {
   },
 
   create: (noticePolicyTemplate, autoSave = true) => {
+    cy.wait(4000);
```
Comment (IhorBohdan) [inline cypress/support/fragments/circulation/newNoticePolicyTemplate.js:43]: Why need here to wait?

File: cypress/support/fragments/circulation/newNoticePolicyTemplate.js [hunk 2]
```diff
@@ -40,7 +40,9 @@ export default {
...
     cy.get('#input-patron-notice-name').type(noticePolicyTemplate.name);
+    cy.wait(5000);
```
Comment (IhorBohdan) [inline cypress/support/fragments/circulation/newNoticePolicyTemplate.js:45]: Why need here to wait?

**Review by IhorBohdan — APPROVED** [review]

**Review by MaksimTerekhov — APPROVED** [review]

File: cypress/support/fragments/circulation/newNoticePolicyTemplate.js (hunk 1 above)
Comment (penfriend) [reply cypress/support/fragments/circulation/newNoticePolicyTemplate.js:43]: this waiter is needed to synchronize with the page.  we need to wait until the field #input-patron-notice-name is shown and ready to receive the text. Without of this waiter only the part of the entered string was saved.

Comment (IhorBohdan) [reply cypress/support/fragments/circulation/newNoticePolicyTemplate.js:43]: Will you add a comment?))

File: cypress/support/fragments/circulation/newNoticePolicyTemplate.js (hunk 2 above)
Comment (penfriend) [reply cypress/support/fragments/circulation/newNoticePolicyTemplate.js:45]: this waiter is needed to synchronize with the page.  we need to wait until the field #input-patron-notice-name is shown and ready to receive the text. Without of this waiter only the part of the entered string was saved.

**Pull request merged**